xml_converter/proto/waypoint.proto

```diff
@@ -21,7 +21,7 @@ message Category {
     repeated Icon icon = 3;
     repeated Trail trail = 4;
     bool is_separator = 5;
-    bool default_visibility = 6;
+    bool hide_category = 6;
```

Comment:
What were other names that were considered here and why did you decide on `hide_category`?
As it is written now this field is `Category.hide_category` which seems redundant.
Just removing `category` and making it `Category.hide` might look like a good idea initially but will it make intuitive sense when combined with the hidden/shown safe file data too?

Reply:
The other top names I considered were `hide_by_default' and `default_hide_category`.
There is another field in the XML called `hide` that we made `trigger.action_hide_category` so I used that naming convention. Though on further thought, that field is a Category object and this one is a bool so maybe they don't need to be similar. 
I considered using `hide` but was concerned that it was too similar to the Godot function `hide()` and that when accessing this field, `Category.get_hide()` isn't descriptive enough.

Reply:
My gut tells me that `hide_category` is too imperative / verb-ish when it should be more declarative / noun-ish.

Reply:
I agree

How about `is_hidden`? This makes it clear its a property not a command. Also fits with some other attributes like `is_separator` and `is_wall`.